### PR TITLE
Preparing 2.1.0

### PR DESCRIPTION
**Pull request by grgrzybek**

### CODE REVIEW

File: src/site/asciidoc/news.adoc [hunk 1]
```diff
@@ -15,6 +15,67 @@
...
+* with the `org.jolokia.service.jmx.handler.list.DataUpdater` Jolokia service, we can get additional information inside JSON-ified `javax.management.MBeanInfo` - 3rd party extensions may add RBAC information or any other data into the MBeanInfo
+
+==== Old issues resolved
+
+It's worth noting that we've also fixed issues dating back up to 2015...
+
+* Added support for custom date format for dates (and new `java.time` classes)
+* Added support for multi-dimensional arrays
+* Added support for `java.util.Date` operation parameters
+* Added ability to declare (in `jolokia-access.xml`) that some MBans simply should not be returned in `search` and `list` operations.
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:75]: MBans --> MBeans

File: src/site/asciidoc/news.adoc [hunk 2]
```diff
@@ -15,6 +15,67 @@
...
+
+* With the new `listKeys` processing/request parameter, `list()` operation returns an additional `"keys"` field for MBean's MBeanInfo - it contains a map of the keys obtained from ObjectName
+* With the new `includeRequest` processing/request parameter, we can tell Jolokia not to return incoming requests under `the "request"` field of the response (defaults to `false`)
+* The' list () ' operation may be optimized with the new `listCache` processing/request parameter. It allows caching the common JSON-ified MBeanInfo to avoid duplicating it in the `list()` response for, say, 50,000 Artemis queue MBeans. Getting 2MB instead of 0.5GB of JSON is always a good thing™. More details can be found in the documentation.
+
+==== New extension points
+
+To make `list()` operation more flexible (based on experience gained from the https://hawt.io/[Hawtio console])), Jolokia now provides extension points for `list()`. While it is still based on `javax.management.MBeanServerConnection.queryMBeans()` call, we don't necessarily blindly serialize each MBean's `javax.management.MBeanInfo`.
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:61]: (based on experience gained from the https://hawt.io/[Hawtio console])) --> (based on experience gained from thehttps://hawt.io/[Hawtio console])

File: src/site/asciidoc/news.adoc [hunk 3]
```diff
@@ -15,6 +15,67 @@
...
+
+==== New Jolokia protocol version
+
+Going with the flow, after moving from JavaScript to TypeScript and getting our own JSON parser, we've introduced a new Jolokia protocol version, `8.0`. There's more information in the link:reference/html/manual/jolokia_protocol.html#optimized-response-list[relevant documentation section], but here's a summary of the changes:
+
+* With the new `listKeys` processing/request parameter, `list()` operation returns an additional `"keys"` field for MBean's MBeanInfo - it contains a map of the keys obtained from ObjectName
+* With the new `includeRequest` processing/request parameter, we can tell Jolokia not to return incoming requests under `the "request"` field of the response (defaults to `false`)
+* The' list () ' operation may be optimized with the new `listCache` processing/request parameter. It allows caching the common JSON-ified MBeanInfo to avoid duplicating it in the `list()` response for, say, 50,000 Artemis queue MBeans. Getting 2MB instead of 0.5GB of JSON is always a good thing™. More details can be found in the documentation.
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:57]: why the ™ ?
I guess I miss a reference

Comment (apupier) [inline src/site/asciidoc/news.adoc:57]: The' list () ' operation --> The \`list()\` operation

File: src/site/asciidoc/news.adoc [hunk 4]
```diff
@@ -15,6 +15,67 @@
...
+
+The new Jolokia module is `org.jolokia:jolokia-json` and it fully replaces `com.googlecode.json-simple:json-simple` library. Class names are the same, the only change is in the package name (switch from `org.json.simple` to `org.jolokia.json`).
+
+==== New Jolokia protocol version
+
+Going with the flow, after moving from JavaScript to TypeScript and getting our own JSON parser, we've introduced a new Jolokia protocol version, `8.0`. There's more information in the link:reference/html/manual/jolokia_protocol.html#optimized-response-list[relevant documentation section], but here's a summary of the changes:
+
+* With the new `listKeys` processing/request parameter, `list()` operation returns an additional `"keys"` field for MBean's MBeanInfo - it contains a map of the keys obtained from ObjectName
+* With the new `includeRequest` processing/request parameter, we can tell Jolokia not to return incoming requests under `the "request"` field of the response (defaults to `false`)
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:56]: under `the "request"` field --> under the `request` field

File: src/site/asciidoc/news.adoc (hunk 2 above)
Comment (grgrzybek) [reply src/site/asciidoc/news.adoc:61]: ah yes - this is important: https://xkcd.com/859/

File: src/site/asciidoc/news.adoc (hunk 3 above)
Comment (grgrzybek) [reply src/site/asciidoc/news.adoc:57]: you're right - I'll remove it.

**Review by rhuss — COMMENTED** [review]
Looks good to me on a quick check (sorry, don't have much time to dig deeper). 

Thanks!

**Review by rhuss — APPROVED** [review]

**Pull request merged**